x-pack/metricbeat/module/sql/_meta/docs.asciidoc

```diff
@@ -22,14 +22,23 @@ fields:
 
 `driver`:: The driver can be any driver that has a {metricbeat} module, such as
 `mssql` or `postgres`.
-`sql_query`:: The single query you want to run.
-`sql_response_format`:: Either `variables` or `table`:
-`variables`::: Expects a two-column table that looks like a key/value result.
+
+`raw_data.enabled`:: Expects either true or false. By default false. Marking as true will generate event results in new field format.
+
+Expects either `sql_query` or `sql_queries`.
+
+`sql_query`:: The single query you want to run. (`Backward Compatibility`)
```

Comment:
If this is the "least" preferred solution, don't put it as the first option on the list of fields to consider.

Reply:
Order changed. There may be some additional doc changes to be added later.